### ISSUE TITLE
2.0.1 not compatible with 2.0.0

### ISSUE THREAD

**Issue by swordbeta** [bernardphp/BernardBundle#51]
Using `DefaultMessage` you now get the following error:

```
UnexpectedValueException: No supported normalizer found for "Bernard\Message\DefaultMessage".

/workspace/vendor/bernard/normalt/src/Normalizer/AggregateNormalizer.php:43
/workspace/vendor/bernard/bernard/src/Normalizer/EnvelopeNormalizer.php:36
/workspace/vendor/bernard/normalt/src/Normalizer/AggregateNormalizer.php:40
/workspace/vendor/bernard/bernard/src/Serializer.php:41
/workspace/vendor/bernard/bernard/src/Queue/PersistentQueue.php:98
/workspace/vendor/bernard/bernard/src/Consumer.php:81
```

Is tihs a bug or am i missing an upgrade guide?

**Comment by acrobat** [comment]
This because of a new release of bernard/bernard. The default message normalizer was moved/renamed to a PlainMessageNormalizer. But for the message class itself we provided some kind of BC layer, but for the normalizer it was not done.

I will add a BC layer for the normalizer and release a new version of bernard tonight!

**Comment by sagikazarmark** [comment]
I would rather add a BC layer in this package and tag a PATCH release instead of tagging another alpha.

**Comment by acrobat** [comment]
@sagikazarmark so maybe just add the DefaultMessageNormalizer in this package as a fallback?

**Comment by sagikazarmark** [comment]
Yup, something like that.

**Comment by acrobat** [comment]
Alright! I will open a PR for this later today!

**Comment by acrobat** [comment]
Alright! I will open a PR for this later today! Should we still include the deprecation notice here in the bundle, so we can remove this fallback later on?

**Comment by sagikazarmark** [comment]
Since the bundle is already stable, I think we should.

**Comment by acrobat** [comment]
@swordbeta I've created a PR for this issue, please test if it solves your issue!

**Comment by acrobat** [comment]
@swordbeta I've released 2.0.2 with a fix for the BC break. Please test and reported any issues!

**Comment by swordbeta** [comment]
@acrobat Working yes, thank you!
